README.md

```diff
@@ -208,6 +187,15 @@ Example:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/torobucci)
+- Twitter: [@twitterhandle](https://twitter.com/KipkuruiKelvin3e)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kelvin-kipkurui-7b50b8252/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] I have noticed that you removed the `MIT.md` file. We should not ignore this file as it gives other developers permission to contribute to your project. As the previous reviewer suggested kindly ensure that the `MIT` link is working properly as intended 👍 
- [ ] [OPTIONAL ] Great job: adding the README file 👏🏻 Let's make it more descriptive, Kindly, it would be nice to add explicit steps for cloning and running your app, as this will help you, non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes git clone URL , cd.., etc.

- [ ] [OPTIONAL ] please remove all unnecessary titles since they are empty or add details to them (Prerequisites, run tests Deployment ....) 🚮
- [ ] The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.